src/Domain.php

```diff
@@ -800,7 +800,8 @@ public static function getUsed(array $used, $domaintype)
     public static function getAdditionalMenuLinks()
     {
         $links = [];
-        if (static::canView()) {
+        $can_manage_records = count($_SESSION['glpiactiveprofile']['managed_domainrecordtypes'] ?? []) > 0;
```

Comment:
ok with the solution.

Can we move the duplicated code into a `static function canManageRecords` method ?